azimuth/modules/model_performance/confidence_binning.py

```diff
@@ -24,23 +28,34 @@
 class ConfidenceHistogramModule(FilterableModule[ModelContractConfig]):
     """Return a confidence histogram of the predictions."""
 
-    def get_outcome_mask(self, outcome: OutcomeName) -> List[bool]:
-        return [utterance_outcome == outcome for utterance_outcome in self._get_outcomes_from_ds()]
-
-    def compute_on_dataset_split(self) -> List[ConfidenceHistogramResponse]:  # type: ignore
-        """Compute the confidence histogram with CONFIDENCE_BINS_COUNT bins on the dataset split.
+    @staticmethod
+    def get_outcome_mask(
+        ds, outcome: OutcomeName, without_postprocessing: bool = False
+    ) -> List[bool]:
+        return [
+            utterance_outcome == outcome
+            for utterance_outcome in get_outcomes_from_ds(ds, without_postprocessing)
+        ]
+
+    @classmethod
+    def get_bins(
+        cls, ds: Dataset, without_postprocessing: bool = False
+    ) -> List[ConfidenceBinDetails]:
+        """Compute the bins on the dataset split.
```

Comment:
```suggestion
        """Compute the bins on the (partial or full) dataset split.
```
Or something else indicating that it's not necessarily the full train or eval splits? (See similar (but broader) comment on metrics.)

Reply:
I'm not certain about this distinction, because all modules can run on partial or full dataset split, the only difference here is that the dataset is sent as an arg, instead of being loaded with `ds = get.dataset_split()`.  Would `Compute the bins on the specified dataset split.` be better? Or something like that?

Reply:
Oh, good point. Sure, that's a little more specific!